Fabric.Authorization.AccessControl/src/app/login/login.component.html

```diff
@@ -1,3 +1,6 @@
-<p>
-  login works!
-</p>
+<div *ngIf="loggingIn">
+  <label>Logging in</label>
+  <i class="fa fa-spinner fa-spin fa-3x fa-fw"></i>
```

Comment:
Is there a reason you did this instead of the Cashmere progress indicator?  I'm using that on the member list page.

Reply:
I can switch it, this login page is not being packaged in the library so I did it as quick as possible